stable/mlrun-kit/templates/minio-persistency/pv.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.minioPersistencyStandalone.pvc.create -}}
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: minio-pv
+  labels:
+    app.kubernetes.io/name: {{ template "mlrun-kit.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+spec:
+  storageClassName: "minio-host-path"
+  capacity:
+    storage: 2Gi
```

Comment:
same ^

Reply:
this is the pv that create the local volume